src/com/encora/javaAssignments/DailyProgramPractice/program2/FindVowels.java

```diff
@@ -0,0 +1,19 @@
+package DailyProgramPractice;
+
+public class FindVowels {
+    //Write a Java method to count all vowels in a string.
+    public static int count_Vowels(String string_One) {
+        int count = 0;
+        for (int index = 0; index < string_One.length(); index++) {
```

Comment:
duplicate code

Reply:
Yes, something happened so looking into it.